rllib/env/multi_agent_episode.py

```diff
@@ -0,0 +1,535 @@
+import uuid
+
+from typing import Any, Dict, List, Optional, Union
+
+from ray.rllib.env.single_agent_episode import SingleAgentEpisode
+from ray.rllib.policy.sample_batch import MultiAgentBatch
+
+
+class IndexMapping(list):
+    """Provides lists with a method to find elements."""
+
+    def find_indices(self, indices_to_find):
+        indices = []
+        for num in indices_to_find:
+            if num in self:
+                indices.append(self.index(num))
+        return indices
+
+
+# TODO (sven, simon): This is still the old _Episode, i.e. without
```

Comment:
Remove this TODO: We are just hours to merging the PPO EnvRunner PR.

Reply:
Yeah this will resolve a lot of TODOs then. Looking forward!